encfs/main.cpp

```diff
@@ -768,9 +772,18 @@ static bool unmountFS(EncFS_Context *ctx) {
     return false;
   } else {
     // Time to unmount!
-    RLOG(WARNING) << "Unmounting filesystem due to inactivity: "
-                  << arg->opts->mountPoint;
-    fuse_unmount(arg->opts->mountPoint.c_str());
-    return true;
+    int rc=1;
+    if (access("/bin/umount", F_OK) != -1)
+      rc = system(("/bin/umount "+std::string(arg->opts->mountPoint)).c_str());
+    else
+      rc = system(("/sbin/umount "+std::string(arg->opts->mountPoint)).c_str());
```

Comment:
I notice encfs4win is a complete fork.  Is encfs not designed to run on Windows as well?  I guess not since it requires a whole fork.